src/player.js

```diff
@@ -558,7 +561,7 @@ export default class Player extends FakeEventTarget {
    * @public
    */
   getVideoElement(): ?HTMLVideoElement {
-    if (this._engine) {
+    if (this._engine && typeof this._engine.getVideoElement === 'function') {
```

Comment:
`getVideoElement` is part of `engine` interface, it must be a function

Reply:
it's not. I added it because there was an error when I checked it without UI engine was exist but `getVideoElement` wasn't